src/test/tests/databases/reopen.py

```diff
@@ -104,7 +107,7 @@ def FilesPresent(files):
 # to ensure that there are no such files left over from a failed test.
```

Comment:
You might want to update this comment to say the run directory instead of the current directory.